src/helper.rs

```diff
@@ -16,3 +17,46 @@ pub struct Node {
     pub udp_port: u16,
     pub node_id: Identifier,
 }
+
+// For testing. TODO: Create mod
+pub fn create_dummy_nodes() -> Vec<Node> {
+    // Should these nodes have different IP addresses?  Stupid question- but I'm asking anyways   :P
+    let listen_addr = String::from("127.0.0.1").parse::<Ipv4Addr>().unwrap();
+    let port_start = 9000_u16;
+    let mut our_nodes = Vec::new();
+
+    let local_node = Node {
+        ip_address: listen_addr,
+        udp_port: port_start,
+        node_id: [0_u8; 32],
+    };
+    our_nodes.push(local_node);
+    let first_node = Node {
+        ip_address: listen_addr,
+        udp_port: port_start + 1,
+        node_id: [1_u8; 32],
+    };
```

Comment:
a bit of a tangent but you could make a single helper fn to help you make these instances

something like:
```rust
fn mk_node(listen_addr: &Ipv4Addr, port_start: u16, index: usize) -> Node {
    Node {
         ip_address: listen_addr.clone(),
        udp_port: port_start + index as u16,
        node_id: [index as u8; 32],
}
```

then something like
```rust
let our_nodes: Vec<_> = (0..4).into_iter().map(mk_node).collect();
```

Reply:
Implemented here:  https://github.com/EchoAlice/my_kademlia/commit/5ebfa6fdb6f4bb5b29b2123e8f1912dc251e6dc2